Wrappers/Python/cil/plugins/ccpi_regularisation/functions/regularisers.py

```diff
@@ -123,7 +134,15 @@ def __init__(self, alpha=1, max_iteration=100, tolerance=0, isotropic=True, nonn
         self.nonnegativity = nonnegativity
         self.device = device # string for 'cpu' or 'gpu'
 
+        super(FGP_TV, self).__init__(strong_convexity_constant=strong_convexity_constant)
+
+
     def proximal_numpy(self, in_arr, tau):
+
+        if self.strong_convexity_constant>0:
+            tau /= (1+tau*self.strong_convexity_constant)
+            in_arr /= (1 + tau*self.strong_convexity_constant)
```

Comment:
personally I find clearer the second line
```python
# this is better 
(1 + tau*self.strong_convexity_constant)
# than this
(1+tau*self.strong_convexity_constant)
```

Reply:
fixed